<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPerson</class>
 <widget class="QWidget" name="addPerson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Enter The Person's Role</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nameorg">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>30</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Enter The Person's Name </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nameorg_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
